server/utils/utils.ts

```diff
@@ -206,6 +206,19 @@ const isPassedArdOrCrd = (licence: LicenceSummary, prisoner: Prisoner | PrisonAp
   return false
 }
 
+const isInPssPeriod = (licence: Licence): boolean => {
+  const { licenceExpiryDate, topupSupervisionExpiryDate } = licence
```

Comment:
I don't think domain logic belongs in utils but it feels a bit like I'm fighting a losing battle :-D 

We should be looking at moving all stuff like this down in to the API layer anyway but I'd suggest leaving this here for now..

Could do with a test though?